packages/components/src/core/DropdownMenu/index.tsx

```diff
@@ -104,41 +102,61 @@ const DropdownMenu = <
         {
           name: "offset",
           options: {
-            offset: [0, 8],
+            offset: [0, theme.app?.spacing.s],
           },
         },
       ]}
       open={open}
       anchorEl={anchorEl}
-      {...PopperBaseProps}
       placement={PopperPlacement}
+      {...PopperBaseProps}
+      sx={{
```

Comment:
not sure how much this matters, but should we extract this in `useMemo()` just in case?

Reply:
Done! 🥸